hack/overlays/clusterrolebindings.yaml

```diff
@@ -0,0 +1,19 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  #@yaml/text-templated-strings
+  name: knative-(@= data.values.namespace @)
+  labels:
+    networking.knative.dev/ingress-provider: contour
```

Comment:
Yes, we can drop it as long as the overlay that adds the label is applied after this one. I'll rename the files and confirm that it is the case.

Reply:
> Can we change the overlay file names to be more descriptive -
> 
> ie. instead of `overlay-clusterrolebindings.yaml` - we just call it `delete-contour-clusterrolebinding.yaml` - then we can drop the comments if needed

I just pushed the changes. Let me know of any suggestions for the file names